packages/devtools_app/test/integration_test_infra/in_file_args_test.dart

```diff
@@ -7,78 +7,41 @@ import 'package:flutter_test/flutter_test.dart';
 import '../../integration_test/test_infra/run/_in_file_args.dart';
 import '../../integration_test/test_infra/run/_test_app_driver.dart';
 
-const _testAppPath = 'test/test_infra/fixtures/memory_app';
+void main() {
+  test('$TestFileArgs, empty, flutter app device', () {
```

Comment:
nit: wrap these tests in a group with the name `'$TestFileArgs'` so that we don't have to repeat that in each test name

Reply:
Done